panel/io/django.py

```diff
@@ -1,7 +1,7 @@
 from contextlib import contextmanager
 from urllib.parse import urljoin, urlparse
 
-from bokeh.server.django.consumers import AutoloadJsConsumer, DocConsumer
+from bokeh_django.consumers import DocConsumer, AutoloadJsConsumer
```

Comment:
@philippjfr I'm wondering if it makes sense to move this code to the `bokeh_django` repo rather than maintaining it here. We could conditionally import `panel` and if present then modify the `DocConsumer` and `AutoloadJsConsumer` accordingly.